packages/jest-reporters/src/CoverageReporter.ts

```diff
@@ -273,7 +273,16 @@ export default class CoverageReporter extends BaseReporter {
         const pathOrGlobMatches = thresholdGroups.reduce<
           Array<[string, string]>
         >((agg, thresholdGroup) => {
-          const absoluteThresholdGroup = path.resolve(thresholdGroup);
+          // Preserve trailing slash, but not required if root dir
+          // See https://github.com/facebook/jest/issues/12703
+          const resolvedThresholdGroup = path.resolve(thresholdGroup);
+          const suffix =
+            (thresholdGroup.endsWith(path.sep) ||
+              (process.platform === 'win32' && thresholdGroup.endsWith('/'))) &&
```

Comment:
```suggestion
            thresholdGroup.endsWith('/') &&
```

same thing, no? `win32` is the only platform where `path.sep` is _not_ `/`

Reply:
Does this still allow for windows style slashes in the thresholdGroup? On windows it first checks for windows style path.sep, then checks for POSIX since windows allows both. On all other platforms only the first part of the OR is used

Reply:
ah no, I misread the condition